doc/README_docker_gnu.md

```diff
@@ -424,8 +421,7 @@ You will see the final job, the post, finish its 48th hour:
     + (( i<=48 ))
 
 
-6. Where is my Output?
-----------------------
+## Section 6: Where is my Output?
```

Comment:
Did we take out the graphics generation from the latest version of the Docker container? When I ran ./run_all.sh it ended with the graphics generation and an informational message about where to view the output.

Reply:
I'd like to keep section 6 in, so people know where the output is even if they close the terminal that had the run_all.sh output.